watertap/ui/fsapi.py

```diff
@@ -286,7 +339,7 @@ def build(self, **kwargs):
         try:
             self.run_action(Actions.build, **kwargs)
         except Exception as err:
-            _log.error(f"Building flowsheet: {err}")
+            raise RuntimeError(f"Building flowsheet: {err}") from err
```

Comment:
Can we keep both?

Reply:
The `RuntimeError` will stop execution and print this message to stderr -- so I'm not sure why we'd want both

Reply:
I'm not sure, but errors in `fsapi` might already be handled at the UI backend level, and therefore would be logged at the `WaterTAP-UI` application layer (@MichaelPesce will know more on this).

That said, I don't have anything against doing both the logging and the raising here, since I think the worst that can happen is a redundant log entry.